chicken-sloths/bangazon-cli-sprint2
# Fixes #63: Cleans up ui.js & does not allow user to input same payment type

# Description
This PR cleans up `app/ui.js` so that `addCustPaymentOpt` follows the same form as the other controllers.
Furthermore, a user cannot add a payment option for a customer if they already have a payment option of that kind. E.g., can't have two 'Visa' payment options.

## Related Ticket(s)
Fixes #63 

## Problem to Solve
`app/ui.js` was a bit of a mess
Customer would have two or payment options of the same type and there was no way to distinguish which was which in closing out an order

## Proposed Changes
`addCustPaymentOptC` calls the different models in one function instead of exporting two functions.
In that prompt, there's a check against that user's available payment options to ensure a duplication can't be added.

## Expected Behavior
User can only add a single payment option for a customer for each payment type

## Steps to Test Solution

1. `npm test`
1. `npm run db:reset`
1. `npm start`
1. Select `2`, and then any customer id
1. Select `3`, and then `18`, enter an account number
1. Repeat previous step, you should get a message saying `Customer already has a payment type of that kind`

## Testing
